interpreter/exec/v128.ml

```diff
@@ -4,6 +4,17 @@ include Simd.Make
     let bytewidth = 16
     let to_string s = s
 
+    let to_i32x4 s =
+      List.map (fun i -> I32.of_bits @@ Bytes.get_int32_le (Bytes.of_string s) i) Simd.f32x4_indices
```

Comment:
Nit: Ocaml's `@@` is confusing with our own use of the operator. We should rename that at some point, but until then I would prefer to avoid using `@@`, esp in cases like this, where it actually makes the line longer.

Reply:
Make sense, fixed!